src/libraries/pretest.proj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.Build.Traversal">
 
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" />
```

Comment:
Don't we need to import the Sdk.targets file from the SharedFramework.Sdk?

Reply:
What does that do? Originally I had imported just `Sdk.targets` to include the taskName `CreateFrameworkListFile`, but that led to some build errors. It was learned that importing both `Sdk.props` and `Sdk.targets` resolved the build errors, and then we omitted `Sdk.targets` upon learning that only `Sdk.props` led to a successful build to remove any superfluous imports.

Reply:
Only importing half of an msbuild sdk usually leads to more errors in the long run. Do you still have the errors when importing the Sdk.targets as well somewhere?

@dagood as you authored the Sdk in Arcade, can you help here?

Reply:
I don't believe we need to import the targets.  It seems like we only need the location of the assembly to load the `CreateFrameworkListFile` task.

Reply:
The sharedfx SDK happens to be well-behaved here (the targets don't redefine any static props you need, unlike the Arcade SDK which sticks a lot of static props in the target files) but to be safe it is better to import both. It is also very unusual not to include both, so if you don't want to import the targets IMO you should leave a comment indicating why.

Reply:
> Do you still have the errors when importing the Sdk.targets as well somewhere?

It sounds to me like there weren't any errors with both, the targets import was just removed since it seemed like unneeded code at the time (which is fair since MSBuild norms are terribly hard to uncover 😛).

Reply:
Done, added the targets import after props import (targets then props had a build error)